packages/electron/prepare-build.js

```diff
@@ -17,6 +17,7 @@ async function main() {
     const { width } = await baseIcon.metadata();
     const padding = Math.floor(width / 20);
     await baseIcon
+        .resize(512 - padding * 2, 512 - padding * 2, { fit: "inside" }) // Some targets don't deal well with images > 512x512
```

Comment:
I _thought_ `.resize(512, 512)` was enough, but the `extend` in the same "pipeline" would make it always increase in size.